app/main/controllers/template/RTMediaNav.php

```diff
@@ -454,4 +456,95 @@ function set_privacy( $profile ) {
 
 		return $privacy;
 	}
+
+	/**
+	 * Check if media component has counts.
+	 *
+	 * @param bool   $count  Component feed has count.
+	 * @param object $object Nav object.
+	 * @param string $value  Current component (personal, groups, etc).
+	 *
+	 * @return bool
+	 */
+	public function rtmedia_bp_nouveau_nav_has_count( $count, $object, $value ) {
+
+		// Check if its media.
+		if ( RTMEDIA_MEDIA_SLUG !== $object->slug ) {
+			return $count;
+		}
+
+		global $bp;
+
+		$media = [];
+
+		// If current component is `groups` then fetch its media.
+		if ( 'groups' === $value ) {
+			$media = $this->actual_counts( $bp->groups->current_group->id, 'group' );
+		}
+
+		// If current component is `personal` then fetch its media.
+		if ( 'personal' === $value ) {
+			$media = $this->actual_counts( $bp->displayed_user->id );
+		}
+
+		if ( empty( $media ) ) {
+			return $count;
+		}
+
+		// Convert to number.
+		$media_count = rtmedia_number_to_human_readable( $media['total']['all'] );
```

Comment:
Should empty check be added for `$media['total']` and `$media['total']['all']`, also since the function expects the param to be int may be use `absint()`